src/www/app/app.ts

```diff
@@ -202,14 +209,12 @@ export class App {
       messageKey = 'error-unexpected';
```

Comment:
Let's always show the error details in this case, so we can capture the error type even if there are no messages or cause.